uxarray/grid/grid.py

```diff
@@ -537,7 +538,7 @@ def edge_lon(self) -> xr.DataArray:
         Dimensions (``n_edge``)
         """
         if "edge_lon" not in self._ds:
-            return None
+            return _populate_edge_centroids(self)
```

Comment:
We shouldn't be returning this function, we just need to call it

```suggestion
            _populate_edge_centroids(self)
```

Can you make this change to all of the ones you've added?